### DISCUSSION TITLE
Safe Exam - problems with starting

### DISCUSSION THREAD

**Discussion by R-A-Hill** [SafeExamBrowser/seb-win-refactoring#445] [Q&A] [answered]
Hi,

Why is this happening when trying to start safe exam? 
![20220810_084440 1](https://user-images.githubusercontent.com/110964613/183833644-da90df7a-daca-4ffd-bb04-bf65d10babfc.jpg)

**Comment by dbuechel** [comment, marked as answer]
You appear to have URL filtering enabled. However, SEB 2.x is not actively maintained anymore, thus we recommend to update to the latest release version of SEB 3.x and reevaluate whether the issue still persists.
